content/en/docs/toolbox/auditing-operating.md

```diff
@@ -0,0 +1,385 @@
+ ResponseStatus.code      | Suggested HTTP return code for the request
+ ResponseStatus.Status    | Status of the operation.
+ RequestReceivedTimestamp | Time the request reached the apiserver.
+ StageTimestamp           | Time the request reached the current audit stage.
+
+ For example, to match all logs in the namespace `test`,
+
+```
+ObjectRef.Namespace = "test"
+```
+
+ to match all logs in the namespaces start with `test`,
+
+```
+ObjectRef.Namespace like "test*"
+```
+
+ to match all logs happened in the latest one hour,
+
+```
+RequestReceivedTimestamp >= "2020-06-12T09:23:28.359896Z" and RequestReceivedTimestamp <= "2020-06-12T10:23:28.359896Z"
+```
+
+#### Macro
+`Macro` is a rule condition snippets that can be re-used inside rules and even other macros. Macros provide a way to name common patterns and factor out redundancies in rules. Here¡¯s an example of a macro.
+
+```
+apiVersion: auditing.kubesphere.io/v1alpha1
+kind: Rule
+metadata:
+  name: alerting-rule
+  labels:
+    workspace: system-workspace
+    type: alerting
+spec:
+  rules:
+  - name: pod
+    type: macro
+    desc: pod
+    macro: ObjectRef.Resource="pods"
+```
+
+ > `Macro` can bu used in rules or other macros like ${pod} or ${alerting-rule.pod}. The difference between these two methods is, ${pod} only can be used in the CRD Rule `alerting-rule`, ${alerting-rule.pod} can be used in all CRD Rule. It is also applied to lists and alias.
+
+#### List
+`List` is collections of items that can be included in rules, macros, or other lists. Unlike rules and macros, lists cannot be parsed as filtering expressions. Here¡¯s an example of a list.
+```
+apiVersion: auditing.kubesphere.io/v1alpha1
+kind: Rule
+metadata:
+  name: alerting-rule
+  labels:
+    workspace: system-workspace
+    type: alerting
+spec:
+  rules:
+  - name: action
+    type: list
+    desc: all operator needs to be audit
+    list:
+      - create
+      - delete
+      - update
+      - patch
+```
+
+#### Alias
+`Alias` is a short name of a filter field, it can be included in rules, macros, lists, and output string. Here¡¯s an example of an alias.
+```
+apiVersion: auditing.kubesphere.io/v1alpha1
+kind: Rule
+metadata:
+  name: alerting-rule
+  labels:
+    workspace: system-workspace
+    type: alerting
+spec:
+  rules:
+  - name: namespace
+    type: alias
+    desc: the alias of the resource namespace
+    alias: ObjectRef.Namespace
+```
+
+#### Output
+ The `Output` string used to format the alert message when the auditing log triggered an alert. The `Output` string can include lists and alias. Here is an example.
+
+```
+Output: ${user} ${verb} a HostNetwork Pod ${name} in ${namespace}.
+```
+ > The field of the user, verb, namespace, name all are alias.
+
+### KubeSphere Auditing Logs Query
+
+ KubeSphere supports auditing logs query for tenant isolation. Use the `admin` account to log in KubeSphere, choose **Toolbox -> Auditing Operating**.
```

Comment:
Is the account admin a must here? I reckon any account can use the query feature for auditing logs as long as the component is enabled in the cluster.